app/helpers/content_helper.rb

```diff
@@ -9,16 +9,58 @@ def block_code(code, language)
     def image(link, title, alt_text)
       image_tag(link, title: title, alt: alt_text, loading: "lazy")
     end
+
+    # losely based on https://github.com/vmg/redcarpet/blob/3e3f0b522fbe9283ba450334b5cec7a439dc0955/ext/redcarpet/html.c#L297
```

Comment:
```suggestion
    # loosely based on https://github.com/vmg/redcarpet/blob/3e3f0b522fbe9283ba450334b5cec7a439dc0955/ext/redcarpet/html.c#L297
```
Minor spelling mistake